hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -89,6 +112,29 @@ const orderSlice = createSlice({
                 payload?.message ||
                 "There was a problem retrieving orders. If this continues please contact hackathon organizers.";
         });
+
+        builder.addCase(updateOrder.pending, (state) => {
+            state.isLoading = true;
+            state.error = null;
+        });
+
+        builder.addCase(updateOrder.fulfilled, (state, { payload }) => {
+            state.isLoading = false;
+            state.error = null;
+            if (payload) {
+                pendingOrderAdapter.updateOne(state, {
```

Comment:
when you cancel an order, it shouldn't show up on the dashboard anymore, you can remove it from the store